README.md

```diff
@@ -22,11 +22,27 @@
 
 # 📖 Gateway Services <a name="about-project"></a>
 
-Gateway is a collection of services that helps make interacting with Frequency as easy as working with any Web2 API!
+Gateway is a collection of services designed to simplify interactions with the Frequency blockchain. It bridges the gap between Web2 and Web3, allowing developers to interact with Frequency as easily as they would with any traditional Web2 API.
```

Comment:
I feel like this isn't very true because of all of the queue systems we have in place. Maybe something closer to "interact with Frequency more like a traditional Web2 API."

Reply:
I can see that but those are more align with traditional Web2 development.